src/javascript/tracking.js

```diff
@@ -4,10 +4,10 @@ import {init as initSession} from './core/session.js';
 import {init as initSend} from './core/send.js';
 import {event} from './events/custom.js';
 import {page} from './events/page-view.js';
-import {init as initClick} from './events/click.js';
+import click from './events/click.js';
```

Comment:
I don't think this works. The `events/click.js` file exports only an `init`. This is currently importing something which does not exist.

Reply:
ah my bad, I read the export as a default export :)